suite/tests/client-interface/thread.dll.c

```diff
@@ -206,6 +222,10 @@ exit_event(void)
     dr_event_destroy(child_alive);
     dr_event_destroy(child_continue);
     dr_event_destroy(child_dead);
+    ASSERT(counter32 == 2);
```

Comment:
Minor: it's not easy to see why it's 2. I guess it _is_ 2 but is it somehow possible to make it clear why, even if it's just a named constant like PARALLEL_THREADS_NUM or sth.

Reply:
Fixed.